test/graphql/mutations/update_profile_test.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# test/graphql/mutations/update_profile_test.rb
+
+require 'test_helper'
+
+module Mutations
+  class UpdateProfileMutationTest < ActionDispatch::IntegrationTest
+    setup do
+      password = 'Password123!'
+      expected_email = 'info@kwartracker.com'
+      @user = create(:user,
+                     email: expected_email,
+                     password: password,
+                     password_confirmation: password)
+      post('/graphql',
+           params: {
+             query: sign_in_with_email_mutation,
+             variables: sign_in_with_email_mutation_variables({ email: @user.email,
+                                                                password: password })
+           })
+
+      @json_response = parse_graphql_response(response.body)
+      @token = @json_response.dig('signInWithEmail', 'token')
```

Comment:
@jaimejazarenoiii , pls remove @ if the variable is used in function only

Reply:
nice catch 👏🏻 thanks